packages/configs/default/index.json

```diff
@@ -26,6 +26,8 @@
     "*.pug": ["@parcel/transformer-pug"],
     "*.coffee": ["@parcel/transformer-coffeescript"],
     "*.mdx": ["@parcel/transformer-mdx"],
+    "webp:*.{jpg,jpeg,png,tiff,webp}": ["@parcel/transformer-webp"],
+    "*.webp": ["@parcel/transformer-raw"],
```

Comment:
This is related to the url thing? How do we default to url?

Is this considered a bug or should we do `url:webp:...`? Seems kinda strange and not sure if that would even work.